apps/web/src/store/modals.ts

```diff
@@ -42,5 +44,8 @@ export const useGlobalModalStateStore = create<GlobalModalState>((set) => ({
   showMobileDrawer: false,
   setShowMobileDrawer: (showMobileDrawer) => set(() => ({ showMobileDrawer })),
   showInvitesModal: false,
-  setShowInvitesModal: (showInvitesModal) => set(() => ({ showInvitesModal }))
+  setShowInvitesModal: (showInvitesModal) => set(() => ({ showInvitesModal })),
+  reportProfileModal: false,
```

Comment:
Rename it to `showReportProfileModal`

Reply:
Still not resolved 🙇🏼

Reply:
I haven't pushed the code yet, I just market it for myself, pushing the change now